tools/eval/datasets/minisupervisely.py

```diff
@@ -0,0 +1,206 @@
+import os
+import cv2 as cv
+import numpy as np
+from tqdm import tqdm
+
+
+class MiniSupervisely : 
+    def __init__(self, root) : 
+        self.root = root
+        self.val_path = os.path.join(root, 'val.txt')
+        self.image_set = self.load_data(self.val_path)
+        self.num_classes = 2
+        self.miou = -1
+        self.class_miou = -1
+        self.acc = -1
+        self.class_acc = -1
+
+
+    @property
+    def name(self):
+        return self.__class__.__name__
+    
+
+    def load_data(self, val_path) : 
+        """
+        Load validation image set from val.txt file
+        Args :
+            val_path (str) : path to val.txt file
+        Returns :
+            image_set (list) : list of image path of input and expected image
+        """
+
+        image_set = []
+        with open(val_path, 'r') as f : 
+            for line in f.readlines() : 
+                image_set.append(line.strip().split())
+
+        return image_set
+    
+    
+    def eval(self, model) : 
+        """
+        Evaluate model on validation set
+        Args :
+            model (object) : PP_HumanSeg model object
+        """
+
+        intersect_area_all = []
+        pred_area_all = []
+        label_area_all = []
+
+        pbar = tqdm(self.image_set)
+
+        for input_image, expected_image in pbar : 
+            pbar.set_description(
+                "Evaluating {} with {} val set".format(model.name, self.name))
```

Comment:
These two lines should be placed outside the for loop since the description does not need to be updated every single iteration.

Reply:
This was referred from [eval script of icdar](https://github.com/opencv/opencv_zoo/blob/6c68bc48c6f96042b29b3425174e431ccac38376/tools/eval/datasets/icdar.py#L37).
I've changed it for this script, should I also change it for icdar?

Reply:
> I've changed it for this script, should I also change it for icdar?

Not in this pull request.